sycl/test-e2e/ESIMD/printf.cpp

```diff
@@ -14,8 +14,8 @@
 // Test using __SYCL_USE_NON_VARIADIC_SPIRV_OCL_PRINTF__ is disabled until ESIMD
 // supports it.
 // FIXME: enable that test
-// RUNx: %{build} -D__SYCL_USE_NON_VARIADIC_SPIRV_OCL_PRINTF__ -o %t_nonvar.out
-// RUNx: %{run} %t_nonvar.out | FileCheck %s
+// RUN: %{build} -D__SYCL_USE_NON_VARIADIC_SPIRV_OCL_PRINTF__ -fsycl-device-code-split=per_kernel -o %t_nonvar.out
```

Comment:
Just wondering, why do we need per_kernel splits?

Reply:
Don't know, I aligned with the previous command line. It passes either way, I can remove it.